autoload/codi/load.vim

```diff
@@ -62,6 +62,10 @@ let s:codi_default_interpreters = {
           \ 'bin': 'R',
           \ 'prompt': '^> ',
           \ 'preprocess': function('s:pp_r'),
+          \ },
+      \ 'clojure': {
+          \ 'bin': ['planck', '--verbose', '--dumb-terminal'],
+          \ 'prompt': '^.\{-}=> ',
           \ }
```

Comment:
I didn't notice it before, but this should really have a comma to be consistent with the rest of the dicts. I can change it myself if you like, but since you're already touching this it may just be easier to change it in this PR.